src/util/util_errors.h

```diff
@@ -167,9 +167,15 @@ enum sssd_errors {
 
     ERR_TLS_HANDSHAKE_INTERRUPTED,
 
+    ERR_CA_DB_NOT_FOUND,
+
     ERR_LAST            /* ALWAYS LAST */
 };
 
+enum sssd_exit_status {
+    EXIT_CA_DB_NOT_FOUND = 50
```

Comment:
There is https://github.com/SSSD/sssd/blob/46843d021376df33455977526e2c79b5c2092890/src/util/util.h#L117
And https://github.com/SSSD/sssd/blob/46843d021376df33455977526e2c79b5c2092890/src/util/child_common.h#L39

`enum` might be better, but having a new place to define exit codes feels inconsistent...

Reply:
I've moved the enumerate to `util.h` and included all existing exit codes there